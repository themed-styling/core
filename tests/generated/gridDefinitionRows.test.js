import 'jest-styled-components'

import { gridDefinitionRows } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(gridDefinitionRows, 'gridDefinitionRows')
