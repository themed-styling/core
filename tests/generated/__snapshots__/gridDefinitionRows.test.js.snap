// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gridDefinitionRows renders 0 1`] = `
Array [
  .c0 {
  grid-definition-rows: 0;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: 0;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: 0;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`gridDefinitionRows renders function 1`] = `
Array [
  .c0 {
  grid-definition-rows: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: has;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: has;
}

<div
    className="c0"
  />,
]
`;

exports[`gridDefinitionRows renders invalid theme path 1`] = `
Array [
  .c0 {
  grid-definition-rows: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`gridDefinitionRows renders negative number 1`] = `
Array [
  .c0 {
  grid-definition-rows: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`gridDefinitionRows renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    grid-definition-rows: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-definition-rows: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-definition-rows: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-definition-rows: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    grid-definition-rows: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-definition-rows: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-definition-rows: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-definition-rows: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    grid-definition-rows: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-definition-rows: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-definition-rows: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-definition-rows: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    grid-definition-rows: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-definition-rows: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-definition-rows: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-definition-rows: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`gridDefinitionRows renders object [breakpoints] 2`] = `
Array [
  @media May {
  .c0 {
    grid-definition-rows: July;
  }
}

@media April {
  .c0 {
    grid-definition-rows: August;
  }
}

@media January {
  .c0 {
    grid-definition-rows: September;
  }
}

@media April {
  .c0 {
    grid-definition-rows: November;
  }
}

@media June {
  .c0 {
    grid-definition-rows: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    grid-definition-rows: July;
  }
}

@media April {
  .c0 {
    grid-definition-rows: August;
  }
}

@media January {
  .c0 {
    grid-definition-rows: September;
  }
}

@media April {
  .c0 {
    grid-definition-rows: November;
  }
}

@media June {
  .c0 {
    grid-definition-rows: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    grid-definition-rows: January;
  }
}

@media April {
  .c0 {
    grid-definition-rows: December;
  }
}

@media January {
  .c0 {
    grid-definition-rows: April;
  }
}

@media April {
  .c0 {
    grid-definition-rows: May;
  }
}

@media June {
  .c0 {
    grid-definition-rows: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    grid-definition-rows: January;
  }
}

@media April {
  .c0 {
    grid-definition-rows: December;
  }
}

@media January {
  .c0 {
    grid-definition-rows: April;
  }
}

@media April {
  .c0 {
    grid-definition-rows: May;
  }
}

@media June {
  .c0 {
    grid-definition-rows: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`gridDefinitionRows renders postitive number 1`] = `
Array [
  .c0 {
  grid-definition-rows: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`gridDefinitionRows renders string 1`] = `
Array [
  .c0 {
  grid-definition-rows: July;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: July;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: June;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: June;
}

<div
    className="c0"
  />,
]
`;

exports[`gridDefinitionRows renders valid theme path 1`] = `
Array [
  .c0 {
  grid-definition-rows: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  grid-definition-rows: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
